src/HealthcareApis/HealthcareApis.Test/HealthcareApisService.Test.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PsModuleName>HealthcareApisFhirService</PsModuleName>
```

Comment:
Should be:
```suggestion
    <PsModuleName>HealthcareApis</PsModuleName>
```

Reply:
It will be HealthcareApisService. I will make the change.

Reply:
Sent a message on Teams about this.

Reply:
Done